blog/_posts/2023-09-06-client-quadlet.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: Run and expose Kubernetes Pod based Application using Quadlet and Inlets
+description: As a Kubernetes user, you can use the same YAML file to expose your application using Inlets
+author: Ygal Blum & Valentin Rothberg
+tags: podman quadlet containers
+author_img: ygal
+#image: /images/2021-09-compose/writing.jpg
+date: 2023-09-06
+---
+
+[Quadlet](https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html) is a new way of running containerized workloads in systemd with Podman. Running Podman in systemd achieves a high degree of robustness and automation since systemd can take care of the lifecycle management and monitoring of the workloads. More advanced features such as [Podman auto updates](https://docs.podman.io/en/latest/markdown/podman-auto-update.1.html) and [custom health-check actions](https://www.redhat.com/sysadmin/podman-edge-healthcheck) make Quadlet a handy tool for modern Edge Computing.
+
+This tutorial is inspired by a previous post from Alex Ellis's on [running inlets with compose](https://inlets.dev/blog/2021/09/09/compose-and-inlets.html). But instead of Compose, we want to show how to deploy inlets via Quadlet and make use of Podman's Kubernetes capabilities. Hence, we are going to run a `.kube` file via Quadlet and Podman.
```

Comment:
I'd invert the order of these two paragraphs.

Reply:
Done